event/event_cache_test.go

```diff
@@ -178,7 +178,7 @@ func TestSubManualClose(t *testing.T) {
 // Test that the system doesn't fail under high pressure.
 func TestSubFlooding(t *testing.T) {
 	NUM_SUBS := 100
-	// Keep the reaper out of evts.
+	// Keep the reaper out of this.
```

Comment:
You did! The code usages are intact, but my refactor accidentally clobbered a bunch of usages of the English word 'this', which was unintentional, and which this corrects.